code/modules/mob/living/carbon/emote.dm

```diff
@@ -34,6 +34,17 @@
 							'sound/misc/clap3.ogg',
 							'sound/misc/clap4.ogg')
 
+/datum/emote/living/carbon/crack
+	key = "crack"
+	key_third_person = "cracks"
+	message = "cracks their knuckles."
+	sound = 'sound/misc/knuckles.ogg'
+
+/datum/emote/living/carbon/crack/can_run_emote(mob/living/carbon/user, status_check = TRUE , intentional)
+	. = ..()

```

Comment:
`return ..()` at the end of the proc instead

Reply:
as you please